email_scraper/src/main/java/EmailScraper.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class EmailScraper {
+
+    private HashSet<String> links;
+    private Queue<String> queue;
+    private HashSet<String> emails;
+    private String domain;
+
+    private static int link_count = 0;
```

Comment:
not sure why this is static when everything else is regular member variables

Reply:
You are right. No need to be static. I removed it.